### PR TITLE
chore: Improve open channel view

### LINKED ISSUE #287: Improve Open Channel
# Open Channel

simulator_screenshot_2DE4292F-F115-457B-89EC-4B2A14E19EE9

- The user is flooded with information.
- The screen could use some icons (e.g. behind the balances)
- Make the p2p endpoint less important but show the most important information more prominent, i.e. the amount you want to lock up. 
- Node name: this can be omitted. 
- Is there a reason why this screen is not consistent with the other screen, i.e. have the balance at the top?
- Idea: change the text to something like: `We only support a single channel with 10101. More advanced usecases will come later`. 
- Add icons to explain what is happening, e.g. something like this: 
image
- Opening a channel should redirect to a success page showing some information, e.g. a tx link to mempool and text saying: `You successfully initiated the channel creation. You will be able to use it once the funding transaction is confirmed`
- In the failure case we should display an error

_Originally posted by @bonomat in https://github.com/itchysats/10101/discussions/273#discussioncomment-4193816_
    

- align the design with what was designed in the wireframe, 
    -  remove node name,
- channel amount has to be most present
-  if time: Add icon and redice text; icon on top, text under, then the channel amount

### PR DESCRIPTION
**Pull request by holzeis**
Still a lot of text, but it's much better organized in my point of view. The text could probably use some more love though.

Screenshot 2022-11-22 at 16 45 20

resolves #287

### CODE REVIEW

File: lib/wallet/open_channel.dart [hunk 1]
```diff
@@ -44,57 +45,105 @@ class _OpenChannelState extends State<OpenChannel> {
...
+              Padding(
+                padding: EdgeInsets.only(left: 20.0, right: 20.0),
+                child: Divider(),
               )
             ]),
+            const SizedBox(height: 10),
+            const Center(child: Icon(Icons.private_connectivity_outlined, size: 80)),
+            // const Center(child: Text("How this works", style: TextStyle(fontWeight: FontWeight.w600, fontSize: 24))),
+            const SizedBox(height: 10),
+            // const Center(child: Text("Before trading non-custodial, we need to establish a secure lightning channel.", style: TextStyle(fontWeight: FontWeight.normal, fontSize: 16))),
```
Comment (bonomat) [inline lib/wallet/open_channel.dart:60]:
```suggestion
```

File: lib/wallet/open_channel.dart [hunk 2]
```diff
@@ -44,57 +45,105 @@ class _OpenChannelState extends State<OpenChannel> {
...
+              Balance(balanceSelector: BalanceSelector.bitcoin),
+              SizedBox(height: 10),
+              Padding(
+                padding: EdgeInsets.only(left: 20.0, right: 20.0),
+                child: Divider(),
               )
             ]),
+            const SizedBox(height: 10),
+            const Center(child: Icon(Icons.private_connectivity_outlined, size: 80)),
+            // const Center(child: Text("How this works", style: TextStyle(fontWeight: FontWeight.w600, fontSize: 24))),
```
Comment (bonomat) [inline lib/wallet/open_channel.dart:58]:
```suggestion
```

File: lib/wallet/open_channel.dart [hunk 3]
```diff
@@ -44,57 +45,105 @@ class _OpenChannelState extends State<OpenChannel> {
...
+                      BoxDecoration(color: Colors.blue, borderRadius: BorderRadius.circular(100)),
+                  child: const Center(
+                      child: Text(
+                    "1",
+                    style: TextStyle(color: Colors.white, fontSize: 20),
+                  )),
+                ),
+                title: const Text("Fund your bitcoin wallet"),
+                subtitle: const Text(
+                    "In order to establish a secure channel between 10101 and you we need to lock some Bitcoin on chain through a Lightning channel.",
```
Comment (bonomat) [inline lib/wallet/open_channel.dart:75]:
```suggestion
                    "In order to establish a Lightning channel between 10101 and you we need to lock some Bitcoin on chain.",
```

File: lib/wallet/open_channel.dart [hunk 4]
```diff
@@ -44,57 +45,105 @@ class _OpenChannelState extends State<OpenChannel> {
...
+                    "2",
+                    style: TextStyle(color: Colors.white, fontSize: 20),
+                  )),
+                ),
+                title: const Text("Enter the channel amount"),
+                subtitle: Column(
+                  crossAxisAlignment: CrossAxisAlignment.start,
+                  children: [
+                    const Text(
+                        "Define the amount of bitcoin you want to lock to the lightning channel. 10101 will add the same amount to the Lightning channel.",
```
Comment (bonomat) [inline lib/wallet/open_channel.dart:94]:
```suggestion
                        "Define the amount of bitcoin you want to lock to the Lightning channel. 10101 will double it for great in-bound liquidity.",
```

File: lib/wallet/open_channel.dart [hunk 5]
```diff
@@ -44,57 +45,105 @@ class _OpenChannelState extends State<OpenChannel> {
...
+                      BoxDecoration(color: Colors.blue, borderRadius: BorderRadius.circular(100)),
+                  child: const Center(
+                      child: Text(
+                    "3",
+                    style: TextStyle(color: Colors.white, fontSize: 20),
+                  )),
+                ),
+                title: const Text("Select your maker"),
+                subtitle: const Text(
+                    "This feature is coming, but in this version you can only trade with 10101.",
```
Comment (bonomat) [inline lib/wallet/open_channel.dart:127]:
```suggestion
                    "This feature is coming, but in the current version you can only trade with 10101.",
```

File: lib/wallet/open_channel.dart [hunk 6]
```diff
@@ -44,57 +45,105 @@ class _OpenChannelState extends State<OpenChannel> {
...
+                      BoxDecoration(color: Colors.blue, borderRadius: BorderRadius.circular(100)),
+                  child: const Center(
+                      child: Text(
+                    "4",
+                    style: TextStyle(color: Colors.white, fontSize: 20),
+                  )),
+                ),
+                title: const Text("Ready!"),
+                subtitle: const Text(
+                    "Hit the open channel button and the channel should be established in a few moments.",
```
Comment (bonomat) [inline lib/wallet/open_channel.dart:143]:
```suggestion
                    "Hit the 'Open Channel' button and the channel will be created in a few moments.",
```

**Review by bonomat — APPROVED** [review]
Very nice :)

**Review by klochowicz — APPROVED** [review]
looks great 👍

**Comment by da-kami** [comment]
I'll work in the changes!

File: lib/wallet/open_channel.dart (hunk 4 above)
Comment (luckysori) [reply lib/wallet/open_channel.dart:94]: Define the amount of bitcoin you want to lock to the Lightning channel. 10101 will double it for great in-bound liquidity.
```suggestion
                        "Enter the amount of bitcoin you want to lock into the Lightning channel. 10101 will double it for great inbound liquidity.",
```

**Review by luckysori — APPROVED** [review]
Very nice improvements!

File: lib/wallet/open_channel.dart (hunk 3 above)
Comment (luckysori) [reply lib/wallet/open_channel.dart:75]: Bitcoin should be lowercase when it refers to the coins themselves imo.

Comment (da-kami) [reply lib/wallet/open_channel.dart]: I like it uppercase, but that's probably the German in me 🤪

**Pull request merged**